.buildkite/core.rayci.yml

```diff
@@ -23,6 +23,19 @@ steps:
     depends_on: corebuild
     job_env: forge
 
+  - label: ":ray: core: workflow tests"
```

Comment:
why move the order?

Reply:
The UI looks a bit nicer when the jobs are sorted by the shard size

<img width="419" alt="Screenshot 2023-10-24 at 10 19 29 AM" src="https://github.com/ray-project/ray/assets/128072568/887d5547-199d-42bf-bf96-a411f91cccc0">